site2/docs/reference-configuration.md

```diff
@@ -215,7 +215,7 @@ internalListenerName|Specify the internal listener name for the broker.<br><br>*
 |brokerClientTlsTrustStorePassword| TLS TrustStore password for internal client, used by the internal client to authenticate with Pulsar brokers ||
 |brokerClientTlsCiphers| Specify the tls cipher the internal client will use to negotiate during TLS Handshake. (a comma-separated list of ciphers) e.g.  [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]||
 |brokerClientTlsProtocols|Specify the tls protocols the broker will use to negotiate during TLS handshake. (a comma-separated list of protocol names). e.g.  [TLSv1.2, TLSv1.1, TLSv1] ||
-|ttlDurationDefaultInSeconds|  The default ttl for namespaces if ttl is not configured at namespace policies.  |0|
+|ttlDurationDefaultInSeconds|The default TTL(time to live) for namespaces if TTL is not configured at namespace policies. When the value is set to `0`, TTL is disabled. It is disabled by default. |0|
```

Comment:
```suggestion
|ttlDurationDefaultInSeconds|The default Time to Live (TTL) for namespaces if the TTL is not configured at namespace policies. When the value is set to `0`, TTL is disabled. By default, TTL is disabled. |0|
```

Reply:
@Huanli-Meng Thank you for your suggestion, updated, PTAL again.